.github/workflows/appimage.yml

```diff
@@ -26,14 +34,14 @@ jobs:
     timeout-minutes: 30
 
     steps:
-    - name: Check out main code
-      if: github.event_name == 'workflow_dispatch' || github.event_name == 'push'
+    - name: Check out code
+      if: inputs.release_type != 'pr'
```

Comment:
[note] casing on 'pr'

Reply:
why do we need this if check if we do the same thing whether or not the release type is `pr`?

Reply:
The difference is that the PR release_type will grab from head of the pull_request: `ref: `${{ github.event.pull_request.head.sha }}`